react/src/components/SessionMetricGraph.tsx

```diff
@@ -171,20 +171,28 @@ const SessionMetricGraph: React.FC<PrometheusMetricGraphProps> = ({
   };
 
   return (
-    <BAICard
-      title={getMetricTitle()}
-      type="inner"
-      styles={{
-        body: {
-          padding: `${token.marginMD}px ${token.marginMD}px ${token.marginXS}px ${token.marginMD}px`,
-        },
+    <Flex
+      direction="column"
+      align="stretch"
+      gap="sm"
+      style={{
+        height: '100%',
+        overflow: 'hidden',
       }}
     >
+      <Flex align="center" style={{ height: 55, marginLeft: 52 }}>
```

Comment:
What is the basis for this size? I'm just asking because it ends with 55 instead of 56.